pkg/network/cloud_network.go

```diff
@@ -102,7 +102,11 @@ func renderCloudNetworkConfigController(conf *operv1.NetworkSpec, bootstrapResul
 		data.Data["HTTP_PROXY"] = os.Getenv("MGMT_HTTP_PROXY")
 		data.Data["HTTPS_PROXY"] = os.Getenv("MGMT_HTTPS_PROXY")
 		data.Data["NO_PROXY"] = os.Getenv("MGMT_NO_PROXY")
-		data.Data["AzureMSIAuthentication"] = os.Getenv("AZURE_MSI_AUTHENTICATION")
+		data.Data["AzureManagedClientID"] = os.Getenv("ARO_HCP_MI_CLIENT_ID")
+		data.Data["AzureManagedTenantID"] = os.Getenv("ARO_HCP_TENANT_ID")
+		data.Data["AzureManagedCertDirectory"] = "/var/run/secrets/azure/"
```

Comment:
can you define the path as a constant and then do
```
data.Data["AzureManagedCertPath"] = filepath.Join(constant, os.Getenv("ARO_HCP_CLIENT_CERTIFICATE_PATH"))
```

ARO_HCP_CLIENT_CERTIFICATE_PATH could maybe change to ARO_HCP_CLIENT_CERTIFICATE_NAME? just a nit